.gitignore

```diff
@@ -39,7 +39,6 @@ yarn-error.log
 # BUCK
 buck-out/
 \.buckd/
-*.keystore
```

Comment:
We should do this as:

```
*.keystore
!debug.keystore
```
That would avoid all keystores except debug.keystore

Reply:
I was wondering if there was a better way - good thinking! I'll make that change now.

Reply:
made the change, going to squash and merge!